addons/udes_stock/models/__init__.py

```diff
@@ -13,3 +13,4 @@
 from . import stock_move
 from . import res_users
 from . import stock_picking_type
+from . import stock_rule
```

Comment:
We've discussed this and come to the conclusion that the push from drop functionality should be put into its own module e.g. "udes_routing".

Reply:
Should this been in udes_open or udes_closed_14/addons ?

Reply:
udes-open please :-)